moonraker/components/wled.py

```diff
@@ -288,52 +282,46 @@ def __init__(self: WLED, config: ConfigHelper) -> None:
 
                 logging.info(f"WLED strip: {name}")
 
-                color_order_cfg: str = cfg.get("color_order", "RGB")
-                color_order = color_orders.get(color_order_cfg)
-                if color_order is None:
-                    raise cfg.error(
-                        f"Color order not supported: {color_order_cfg}")
+                # Discard old color_order setting, always support 4 color strips
+                _ = cfg.get("color_order", "")
```

Comment:
The ConfigHelper's various `get` methods accept a `deprecate` keyword argument that you can set to True .   When this flag is set Moonraker will add a warning.  Mainsail users will see this warning in the UI.  I'm not sure if other clients present warnings, but they are all logged as well so users may see it eventually.  So it this line could be something like
```python
_ = cfg.get("color_order", "", deprecate=True)
```

Reply:
Thanks, didn't know about that (but didn't look for it either). Added as suggested, also added information on deprecation to configuration.md.